### ISSUE TITLE
[Feature] Is it possible to swap the crates in dungeons? 

### ISSUE THREAD

**Issue by KhriszB** [DaFuqs/LootCrates#39]
I want to change all dungeons ( And Dungeons+ dungeons )
by changing the default crates with LootCrates.
Also want to do it with mineshaft minecarts. (I don't care if instead of minecarts there will be only a chest.)
Or with every single Minecraft crates which generating randomly.
Is it possible somehow? I tried everything I can think of, but maybe Ik dumb. If it's possible using datapacks then please point me where to start.
Thanks!

If it's not possible then I would be really happy if you could put this option into the mod. (Swap every vanilla loot)

Edit: IDK why Enchantment is the label and how to change it. :I

**Comment by KhriszB** [comment]
Hmmm... Maybe some of my mods break it couse in another instance it works. Ruined portal had an epic chest for example.
I just can't make it woork with some of my mods :(

**Comment by DaFuqs** [comment]
Yes, it should totally work, just checked again in a default instance with just LootCrates added. Maybe you can find out what triggers the incompatibility?

**Comment by KhriszB** [comment]
> Yes, it should totally work, just checked again in a default instance with just LootCrates added. Maybe you can find out what triggers the incompatibility?

Yeah, thank you for reading my messy comments. 
I have multiple moods that adds or changes vanilla loots. 
I try to add them by hand. But cvan you please help me how to start?
As you can see Im a real begginer. 

For example start with modded structures:
terralith:underground/sunken_tower
This structure have a barrel with different loots.
The only thing I want is to change this barrel with (for example) a locked rare barrel.

"loot_table": "minecraft:barrels/terralith:underground/sunken_tower",

```
"entries": [{
			"crate_variant": "ghost_loot_barrel",
			"tracked_per_player": true,
			"lock_mode": "locked",
			"inventory_deletion_mode": "never",
			"replenish_time_scale": "never"
		}]
	},
```

	
I put this (and many other rattempts) in the LootCratesWorldgenSettings.json5 file but I only end up with an empty normal barrel. What I need to do to make it work? 
Thank you so much for the answer!
I love this mod btw
